contrib/patterns/_grid-media.scss

```diff
@@ -0,0 +1,16 @@
+$medium-screen: 1000px;
+$medium: "only screen and (min-width:#{$medium-screen})";
```

Comment:
I have updated the PR with this function. It actually ended up being pretty trivial.